src/main/java/me/earth/earthhack/impl/gui/click/component/impl/ListComponent.java

```diff
@@ -7,12 +7,12 @@
 import me.earth.earthhack.impl.managers.Managers;
 import me.earth.earthhack.impl.util.render.RenderUtil;
 
-public class ListComponent extends Component
+public class ListComponent<M extends Nameable> extends Component
```

Comment:
Do you think you could create a generic Component? Right now we have the most idiotic solution of them all, with instanceof checks for every single type of setting like wtf, wasnt me. The first step has been done here: https://github.com/3arthqu4ke/3arthh4ck/blob/main/src/main/java/me/earth/earthhack/impl/gui/click/component/SettingComponent.java If we could replace all components with that we could just use getSetting() instead of getListSetting, getStringSetting, getBooleanSetting blablabla